src/main/java/Freedom.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Freedom {
+    protected static Task[] storage = new Task[100];
+    static int lastIndex = 0;
+
+    public static void main(String[] args) {
+        String logo = "\t_________________________________________\n";
+        String message = """
+                \tHello! I'm Freedom
+                \tWhat can I do for you?
+                """;
+        String closing = "\tBye. Hope to see you again soon!\n";
+
+        System.out.println(logo + message + logo);
+
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            String line = in.nextLine();
+
+            if (line.equals("bye")) {
+                break;
+            }
+            boolean isCommand = handleInput(line);
```

Comment:
I would suggest using `command` instead of `isCommand` for the input variable. You can  always check if the `command` is empty later on, is just considering it as a boolean may not be the best option